make/common/modules/LauncherCommon.gmk

```diff
@@ -122,9 +122,11 @@ define SetupBuildLauncherBody
     ifeq ($(STATIC_BUILD), true)
       $1_LDFLAGS += -exported_symbols_list \
               $(SUPPORT_OUTPUTDIR)/build-static/exported.symbols
+      # SapMachine 2024-02-16: Add dt_filesocket implementation
       $1_LIBS += \
           $$(shell $(FIND) $(SUPPORT_OUTPUTDIR)/modules_libs/java.base -name "*.a") \
           $(SUPPORT_OUTPUTDIR)/modules_libs/jdk.jdwp.agent/libdt_socket.a \
+          (SUPPORT_OUTPUTDIR)/modules_libs/jdk.jdwp.agent/libdt_filesocket.a \
```

Comment:
Isn't a `$` here missing?

Reply:
I would guess so. Since it is only used in a static build. the error never materialized.